lib/elixir/lib/integer.ex

```diff
@@ -128,8 +128,13 @@ defmodule Integer do
 
   defp do_undigits([], _base, acc),
     do: acc
-  defp do_undigits([digit | tail], base, acc) when is_integer(digit),
-    do: do_undigits(tail, base, acc * base + digit)
+  defp do_undigits([digit | tail], base, acc) when is_integer(digit) do
+    if digit >= base do
+      raise ArgumentError, "invalid digit '#{digit}' in base #{base}"
```

Comment:
@whatyouhide I was referring to the error in same module for invalid base: https://github.com/gmile/elixir/blob/5b228d1076cbbe9dcb23eb357911589f135761a2/lib/elixir/lib/integer.ex#L191.
:bowtie: